docs/RELEASE.md

```diff
@@ -0,0 +1,227 @@
+# Releasing binaries for Gazebo ROS packages
+
+## Official repo
+
+The release of the official gazebo_ros_pkgs from this repository and the
+use of the ROS buildfarm to generate the binaries is well covered by the
+[ROS documentation](http://wiki.ros.org/bloom/Tutorials/ReleaseCatkinPackage).
+
+## Unofficial gazebo versions
+
+### Background
+
+Each ROS release define one version of Gazebo supported officially through
+all the ROS packages. The different combinations of ROS <-> Gazebo can be
+found in the [REP-3](http://www.ros.org/reps/rep-0003.html). Some examples:
+
+ * ROS Kinetic: Gazebo7
+ * ROS Lunar: Gazebo7
+ * ROS Melodic: Gazebo9
+
+Some use cases require the use of alternative combinations of ROS and Gazebo
+versions. The gazebo_ros_pkgs code is usually prepared to be compatible with
+different versions of Gazebo specially the latest ones.
+
+To release a modified version of gazebo_ros_pkgs which support a different
+major version of gazebo, before running bloom some actions needs to be done:
+
+ 1. Make the system to resolve ubuntu package gazebo as gazeboX
+ 2. Fork and modify the [gbp -release repo](https://github.com/ros-gbp/gazebo_ros_pkgs-release)
+ 3. Run bloom on the modified release repository.
+ 4. Custom infrastructure to create .deb packages
+
+### Upstream versions released using this tutorial
+
+The `gbp -release repository` hosts the latest version released by the maintainers
+of `gazebo_ros_pkgs`. When using these instructions to release a new custom version
+the version of `gazebo_ros_pkgs` released will be the latest one existing in the
+official `gbp -release repository`. The version would be the same but the release number
+will be one above since bloom needs to be run on top of the modified `-release repository`.
```

Comment:
There's a lot of valuable info in this paragraph, just to be sure I understood it all: when using a custom `gbp` repo, the version released will still come from the official repository, but the debs will be built against a different Gazebo version, correct?

So for example, if the latest `gazebo_ros_pkgs` official release for Kinetic (Gazebo7) are of version N.M-1, following these instructions one would be able to release N.M-2, which supports Kinetic and Gazebo9. If that's correct, mind adding a more concrete example like this to the paragraph so it's a bit easier to parse for beginners like me?